tasks/start/start_action.js

```diff
@@ -1,10 +1,21 @@
 const execFileSync = require('child_process').execFileSync;
+const { existsSync } = require('fs');
+const { resolve, join } = require('path');
 
 module.exports = function (plugin, run, options) {
   options = options || {};
+  let args = [];
+  let cmd;
+  const relativeScript = join('scripts', 'kibana.js');
+  const absoluteScript = resolve(plugin.kibanaRoot, relativeScript);
+  if (existsSync(absoluteScript)) {
+    cmd = 'node';
+    args.push(relativeScript);
+  } else {
```

Comment:
Is this trying to fall back to the "old way" of starting Kibana? Why don't we just remove it completely and cut a new major version?

Reply:
I ultimately left it because this change is only going into Kibana 7.0+ and I didn't  want cases where we update this repo but can't get the changes back to  Kibana 6.x.  I don't know how likely that is though, I'm alright with removing it.  

Thoughts with that extra info?

Reply:
I think it's fine to treat this repo like we treat the upstream repo, and maintain a `7.x` branch to backport things into for use against < 7.0 versions of Kibana. That way master continues to track master, but we have a place to put fixes while we continue to develop on the `6.x` branch in Kibana. This repo doesn't change much, and continuing to use `^7.0.0` in our plugins in the `6.x` branches upstream should be fine. 

Alternatively, we could open an `8.x` branch here that drops support for the old way in preparation for the future `8.0.0` release of this tool. That way `7.x` of the plugin helpers will continue to work on both, and some time in the future we have a clean way to cut a breaking version that removes legacy code.

My preference would be to just make the breaking change and cut a new major version, but I'm good either way.